roboteam_ai/src/skills/Receive.cpp

```diff
@@ -14,7 +15,7 @@ Receive::Receive(string name, bt::Blackboard::Ptr blackboard) : Skill(std::move(
 
 void Receive::onInitialize() {
     readyToPassSet = false;
-   // basicGtp.setCanMoveInDefenseArea(false);
+    canMoveInDefenseArea=properties->getBool("canMoveInDefenseArea");
```

Comment:
this boolean should actually be dependent on refstate!

Reply:
Could be, but we could also manually set it in the relevant trees that change based on refState regardless.

Reply:
I think that is probably easier.

Reply:
but also more prone to errors. Just keep it for now, in another branch I am redesigning the application of rules anyway